crates/core/src/commands/config.rs

```diff
@@ -158,6 +158,10 @@ pub struct ConfigOptions {
     /// tolerated. Default if not set: larger packfiles are always tolerated.
     #[cfg_attr(feature = "clap", clap(long, value_name = "PERCENT"))]
     pub set_max_packsize_tolerate_percent: Option<u32>,
+
+    /// Do an extra check by decompressing/decrypting all data before uploading to the repository
+    #[cfg_attr(feature = "clap", clap(long))]
+    pub set_extra_check: Option<bool>,
```

Comment:
Set as a default even here?

```suggestion
    /// Do an integrity_check by decompressing/decrypting all data before uploading to the repository
    #[cfg_attr(feature = "clap", clap(long, default = Some(True)))]
    pub set_integrity_check: Option<bool>,
```

Reply:
I think we should overwork this when tackling https://github.com/rustic-rs/rustic/issues/908